### ISSUE TITLE
Homepage email sign up not showing focus ring on error

### ISSUE THREAD

**Issue by qjac** [department-of-veterans-affairs/va.gov-cms#23283]
## Describe the defect
When the email sign up form errors (on blur from the input AND when clicking submit) focus moves to the `va-text-input` wrapper, but no focus ring displays.

## To Reproduce
1. Go to https://www.va.gov/
2. Scroll down to the email signup block, just before the footer.
3. Leave it blank or enter an invalid email. Click submit.
4. See error message display without focus ring visible

## Recommendation
We are not matching VADS error handling currently. The component may update when the underlying VADS component does, but in case it doesn't due to previous customization, this is a low-effort fix to get the current implementation to show the focus ring correctly.

When focus moves to the `va-text-input` wrapper, set a tabindex='-1' on the `va-text-input` wrapper.

## AC / Expected behavior
- [ ] When the email sign up form errors, it is clear where focus is because the focus ring is visible. 

## Screenshots
Showing just after submit is clicked. Error message fires and focus moves, but you can't see where focus moves to:
<img width="614" height="299" alt="Image" src="https://github.com/user-attachments/assets/48ed8d81-3727-4bf7-8384-bfc2c1522cc8" />

 ## Additional context
- [Relevant slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1769718858524349)
- Ticket where this was originally added: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19516
- [VADS error handling](https://design.va.gov/components/form/#error-handling)
- [Draft screenreader error message guidelines](https://dsva.slack.com/archives/C8E985R32/p1769615346720479?thread_ts=1769034005.599099&cid=C8E985R32)
